src/frontend/scss/components/templates/richie/_multiple-columns.scss

```diff
@@ -0,0 +1,399 @@
+// Multiple columns templates stylesheet
+//
+// This aims to only adjust every possible plugins so they correctly fit since they
+// were done for 100% size only. No colour, font or anything else should be changed
+// here.
+//
+
+.multiple-columns {
+  $cell-gutter: 0.5rem;
+
+  @include make-container();
+  @include make-container-max-widths();
+  display: flex;
+  padding: 0;
+  flex-direction: row;
+  flex-wrap: wrap;
+  background: r-theme-val(body-content, base-background);
+
+  @if $body-padding-fix {
+    @include content-padding-fix;
+  }
+  @include media-breakpoint-up(lg) {
+    padding: 1rem;
+  }
+
+  // Impose tiny horizontal padding to every section row
+  .section__row {
+    padding: $cell-gutter;
+  }
+
+  // ----------
+  // Shared base adjustments for short columns (25+33+50)
+  // ----------
+  &__short {
```

Comment:
I'm not a big fan of using `not()` at this tree level, this pseudo class has some caveats from what i remember. Especially you cannot combine multiple `not()` correctly since it is still marked as an experimental feature.

Reply:
The basic usage of `:not` is well supported and combine several `:not` too.
The experimental feature is the use of an argument list within `:not`.

Reply:
Yep but those two ones are distinct things, the first is somewhat like a "AND", the second is a "OR".

Reply:
I will pass on this change, since i'm really not confident with implications with not() usage, and it would need more time to r&d to ensure this won't have any consequences/complications.